crates/ruff_cli/src/printer.rs

```diff
@@ -439,3 +482,29 @@ fn print_fix_summary(writer: &mut dyn Write, fixed: &FxHashMap<String, FixTable>
     }
     Ok(())
 }
+
+/// Build the displayed fix status message depending on the types of the remaining fixes.
+fn display_violations(safe_remaining: i32, unsafe_remaining: i32) -> String {
```

Comment:
The count cannot be negative.

```suggestion
fn display_violations(safe_remaining: u32, unsafe_remaining: u32) -> String {
```